[FIX] Improve the design of readme file and add contributors section.

Freakyab opened the pull request:
[FIX] Improve the design of readme file and add contributors section. 

Resolve issue 
#53

gauravsingh1281 commented:
@Freakyab Show me the screenshot

Freakyab commented:
> @Freakyab Show me the screenshot

https://github.com/Freakyab/Rentalog-Rent-Management-Web-App-For-Landlords/blob/main/README.md

gauravsingh1281 commented:
> > @Freakyab Show me the screenshot
> 
> https://github.com/Freakyab/Rentalog-Rent-Management-Web-App-For-Landlords/blob/main/README.md

Don't change the text or information which are written in the original readme file. Try to redesign it and make it more catchy or engaging.

Freakyab commented:
> > > @Freakyab Show me the screenshot
> > 
> > 
> > https://github.com/Freakyab/Rentalog-Rent-Management-Web-App-For-Landlords/blob/main/README.md
> 
> Don't change the text or information which are written in the original readme file. Try to redesign it and make it more catchy or engaging.

for making it catchy, I have added keywords otherwise the content is same. Also added Contributors names to readme file.

gauravsingh1281 reviewed (changes requested):
This text are different from original one

gauravsingh1281 commented on README.md:
```diff
@@ -1,76 +1,85 @@
-<p>Rentalog is released under the <strong>MIT License</strong>. By contributing to this project, you agree to provide your contributions under the same license.</p>
+# Rentalog - Rent Management Web App For Landlords🌐
+
+<p>Welcome to Rentalog, the avant-garde web application designed to elevate the way landlords manage their rental properties. Rentalog empowers landlords by seamlessly organizing crucial information about renters, ensuring a secure and organized repository for quick and easy access. Beyond tenant data, Rentalog simplifies rent collection and tracking, providing landlords with real-time insights i ...
```
this

gauravsingh1281 commented on README.md:
```diff
@@ -1,76 +1,85 @@
+
+---
+
+## Features 🚀
```
this

gauravsingh1281 reviewed (changes requested):
don't touch the text which is written in the original file

gauravsingh1281 commented:
closes via #106
